### ISSUE TITLE
SEO

### ISSUE THREAD

**Issue by delahee** [HaxeFoundation/Project-Management#15]
We have to get rid of that roast image (https://www.google.fr/search?q=haxe+image&safe=off&rlz=1C1CHFX_frFR562FR564&espv=2&biw=1587&bih=861&source=lnms&tbm=isch&sa=X&ei=FGBvVba4A8r1ULSggfgH&ved=0CAYQ_AUoAQ#safe=off&tbm=isch&q=haxe)

So we have to campaign all ahxe related sites with adequate SEO actions, i ll centralize here required steps to make everything ok.

**Comment by Merelleya** [comment]
You will never get rid of the roast. It is cultural heritage.Also, Image optimization is not really the issue (go google Rust...just to see what you get).

Also, the Haxe sites are actually pretty well done in terms of SEO. The main thing is more editorial content and maybe some extended metadata for enriched SERPs

**Comment by delahee** [comment]
DOH..well I still can given a go ;)

**Comment by jasononeil** [comment]
I agree with Fiene...

Using a google IMAGE search was a cheap shot from Daniel's talk. We
absolutely dominate the first page of Google WEB results for "haxe", even
if you are using private browsing.

I'm sure there is a little more we can do but I do not think it is the
disaster that was implied.

Jason

**Comment by delahee** [comment]
well yes and no, don't take it as an attack with the wonderful work you made. 

We are already first in web Seo, i don't see why we could not be better ranked in "images" as there are SEO for image specifics. And especially since the first image actually comes from a post that was speaking about Haxe the compiler. 

I'll dig in a little ;)

**Comment by Simn** [comment]
I think you should come over and have a nice German Haxe. ;) This is like calling a programming language Baguette and trying to dominate the image results for it.

If anything the problem is that there aren't any interesting images for our Haxe. Also to be frank, I'd rather look at a nice roast than that fifty shades of orange thing we call the Haxe logo any day.

**Comment by ncannasse** [comment]
Lars posted this on twitter
https://t.co/ISvpK3wVYZ
https://t.co/jP3A7zDg1n

I think that search results are much more important than image result for Haxe, let's not spend our precious time on things that doesn't matter. If we improve communication around Haxe, we will eventually have more images in the search results. Until then, let's focus on what's important.

**Comment by ncannasse** [comment]
PS : feel free to close if you agree

**Comment by Simn** [comment]
I actually looked for Rust images during Daniel's talk and almost called him out on that but managed to refrain from doing so.

I would like to keep this issue open as a reminder to look into some SERPs. @Merelleya told me about how that can make look google results much better.

**Comment by elliott5** [comment]
I'm with Simn, I think having a language with the same name a nice German roast makes it memorable - probably an advantage actually, think of Haxe and your mouth waters... mine does!

**Comment by Merelleya** [comment]
I actually ran the site through an onpage analysis tool yesterday and it came out pretty well. There are some structural things like h1/h2/3 and metadata where there is a little room for improvement and I would like to go through that with you @jasononeil but other than that there is no immediate cause for concern on a purely SEO/technical side.

I will have to have another look at the Webmaster tool and see if everything checks out fine there, too.

During these last two years, the focus of SEO has shifted a lot from technical optimization to focusing on great content. In that context, I am, for instance, super pleased about the manual and I think the showcases and any other editorial content we can come up with will be a huge bonus. Using extended metadata (something I did not have the time to talk to Jason about during the Con), we can even indicate that certain posts are to be classified as "news" and so on.

In SEO, its often about having a good, clean technical setup and then choosing your battles. If we were selling shoes or lifestyle products or any other physical product, really, I would definitely recommend extended image optimization and google shopping campaigns and so on. But in this case I think the time is much better spent elsewhere. Also, one usually starts fighting the battles one is most likely to win, and the image search just isn't it.

I will leave this open for another day to let people, who still have something to say, put in their two cents. After that, I will transfer the actionable things (Metadata for SERPs, On-page structure) to other issues so they can be worked on in a more focused manner.
